lib/screens/settings/account/account_settings_delete.dart

```diff
@@ -87,9 +162,7 @@ class AccountSettingsDelete extends StatelessWidget {
                 ),
               ),
               FlatButton(
-                onPressed: () {
-                  deleteUserAccount(context);
-                },
+                onPressed: () => deleteUserAccount(context),
```

Comment:
Clicking on the **Delete** button should pop up another dialog box that lets the user know that their account has been deleted for ex: "Your account has been successfully deleted." I believe I implemented this on my old task before PE-41. It seems like you changed it up a bit that its not showing the dialog box anymore nor popping to the login page after deleting.

Reply:
@ayeejane In your PE-41, there is no implementation of notifying the user their account was successfully deleted. In addition,
users being redirected to login screen implies their account has been deleted. I would not recommend another dialog box popup, based off the current workflow. The user is aware that their account is going to be deleted based off the instructions.